### DISCUSSION TITLE
Module not found: Can't resolve '/my-website/node_modules/@docusaurus/core/node_modules/@babel/runtime/helpers/esm/extends' in '/my-website/node_modules/@docusaurus/theme-classic/lib-next/theme/CodeBlock'

### DISCUSSION THREAD

**Discussion by wjq990112** [facebook/docusaurus#4615] [Q&A] [answered]
Yestoday, I use `yarn upgrade` to upgrade my dependencies.

After upgrade, start via `yarn start`, it throw this error:

```bash
./node_modules/@docusaurus/theme-classic/lib-next/theme/CodeBlock/index.js
Module not found: Can't resolve '/my-website/node_modules/@docusaurus/core/node_modules/@babel/runtime/helpers/esm/extends' in '/my-website/node_modules/@docusaurus/theme-classic/lib-next/theme/CodeBlock'
```

How can i fixed it?

**Comment by slorber** [comment, marked as answer]
Have you tired to delete node modules and reinstall everything or run
docusaurus clear?

**Reply by wjq990112** [reply]
I have tired delete node_modules and reinstall, but it does not work.

Fortunately, after running `docusaurus clear`, it works successfully.

Thank you so much!

**Comment by redhat123456** [comment]
I&nbsp; don ' t konw&nbsp;


but I think of you can&nbsp;try use cmd again to&nbsp; clone hexo-theme&nbsp;


I hope know help&nbsp; you&nbsp;&nbsp;


see&nbsp; you again !!!!!&nbsp; &nbsp; &nbsp;&nbsp;




***@***.***




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月15日(星期四) 凌晨0:48
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [facebook/docusaurus] Module not found: Can&#39;t resolve ***@***.******@***.***/runtime/helpers/esm/extends&#39; in ***@***.***/theme-classic/lib-next/theme/CodeBlock&#39; (#4615)





 
Yestoday, I use yarn upgrade to upgrade my dependencies.
 
After upgrade, start via yarn start, it throw this error:
 .***@***.***/theme-classic/lib-next/theme/CodeBlock/index.js Module not found: Can't resolve ***@***.******@***.***/runtime/helpers/esm/extends' in ***@***.***/theme-classic/lib-next/theme/CodeBlock'
 
How can i fixed it?
